lib/resources/forms.js

```diff
@@ -181,7 +181,7 @@ module.exports = (service, endpoint) => {
       .then(([form, project]) =>
         Promise.all([
           auth.canOrReject('form.update', form),
-          auth.canOrReject('dataset.list', project)
+          auth.canOrReject('dataset.read', project)
```

Comment:
Same question here about whether the diff might include multiple datasets in the future.